scripts/deployment/deployTokenBridgeCreator.ts

```diff
@@ -30,28 +34,59 @@ const ARB_GOERLI_WETH = '0xEe01c0CD76354C383B8c7B4e65EA88D00B06f36f'
  * @param l2Url
  * @returns
  */
-export const deployTokenBridgeCreator = async (rollupAddress: string) => {
-  if (envVars.baseChainRpc == undefined)
+export const deployTokenBridgeCreator = async () => {
+  if (!envVars.baseChainRpc || envVars.baseChainRpc == '') {
     throw new Error('Missing BASECHAIN_RPC in env vars')
-  if (envVars.baseChainDeployerKey == undefined)
+  }
+  if (!envVars.baseChainDeployerKey || envVars.baseChainDeployerKey == '') {
     throw new Error('Missing BASECHAIN_DEPLOYER_KEY in env vars')
-  if (envVars.childChainRpc == undefined)
-    throw new Error('Missing ORBIT_RPC in env vars')
+  }
+  if (!envVars.baseChainWeth || envVars.baseChainWeth == '') {
+    throw new Error('Missing BASECHAIN_WETH in env vars')
+  }
+  if (
+    (!envVars.rollupAddress || envVars.rollupAddress == '') &&
+    (!envVars.childChainRpc || envVars.childChainRpc == '') &&
+    !envVars.gasLimitForL2FactoryDeployment
+  ) {
+    throw new Error(
+      'Either GAS_LIMIT_FOR_L2_FACTORY_DEPLOYMENT or (ROLLUP_ADDRESS and ORBIT_RPC) must be set in env vars'
```

Comment:
This check doesnt quite match the error message - maybe it should be:
`!(envVars.rollupAddress && envVars.childChainRpc) && !envVars.gasLimitForL2FactoryDeployment`

Reply:
fixed